etl/steps/data/garden/gapminder/2023-09-18/under_five_mortality.meta.yml

```diff
@@ -0,0 +1,114 @@
+tables:
+  under_five_mortality:
+    title: Child Mortality Rate (Gapminder)
+    description: |
+      "This data has been compiled by Klara Johansson and Mattias Lindgren (Gapminder) from a selection of sources:
+      * Human Mortality Database
+      * Child Mortality Estimates from the UN Inter-agency Group for Child Mortality Estimation.
+      * Gapminder model based on infant mortality ratio (version 2) https://www.gapminder.org/data/documentation/gd002/
+      * Model estimates based on Gapminder's life expectancy data combined with model life tables, with some additional adjustments
+
+      We do not include rows marked by Gapminder as 'Guesstimates', which are extrapolations."
+    variables:
+      under_five_mortality:
+        title: Under five mortality
+        origins:
+          - producer: Gapminder
```

Comment:
You don't need to define origins here. They should be propagated automatically. If you did it because you couldn't pinpoint where the propagation failed, I can help you out.